include/aspect/utilities.h

```diff
@@ -989,7 +989,7 @@ namespace aspect
     double compute_spd_factor(const double eta,
                               const SymmetricTensor<2,dim> &strain_rate,
                               const SymmetricTensor<2,dim> &dviscosities_dstrain_rate,
-                              const double safety_factor);
+                              const double C_safety);
```

Comment:
I think you can keep calling things here `SPD_safety_factor`. That also matches the name of the variable you pass in, which makes it easier to read.